edx_sailthru/sailthru_content/sync_library.py

```diff
@@ -96,7 +96,7 @@ def load_fixups(filename):
     if not filename:
         return []
 
-    with open(filename, 'rb') as f:
+    with open(filename, 'rt') as f:
```

Comment:
@schenedx you shouldn't need either of these if you're just trying to read the file. `open(filename) as f:` would be enough.